packages/medusa/src/api/routes/admin/regions/index.ts

```diff
@@ -85,21 +85,81 @@ export const defaultAdminRegionRelations = [
   "fulfillment_providers",
 ]
 
+/**
+ * @schema AdminRegionsRes
+ * type: object
+ * properties:
+ *   region:
+ *     $ref: "#/components/schemas/Region"
+ */
 export class AdminRegionsRes {
   region: Region
 }
 
+/**
+ * @schema AdminRegionsListRes
+ * type: object
+ * properties:
+ *   regions:
+ *     type: array
+ *     items:
+ *       $ref: "#/components/schemas/Region"
+ *   count:
+ *     type: integer
+ *     description: The total number of items available
+ *   offset:
+ *     type: integer
+ *     description: The number of items skipped before these items
+ *   limit:
+ *     type: integer
+ *     description: The number of items per page
+ */
 export type AdminRegionsListRes = PaginatedResponse & {
   regions: Region[]
 }
 
+/**
+ * @schema AdminRegionsDeleteRes
+ * type: object
+ * properties:
+ *   id:
+ *     type: string
+ *     description: The ID of the deleted Region.
+ *   object:
+ *     type: string
+ *     description: The type of the object that was deleted.
+ *     default: region
+ *   deleted:
+ *     type: boolean
+ *     description: Whether or not the items were deleted.
+ *     default: true
+ */
 export type AdminRegionsDeleteRes = DeleteResponse
 
 export class FulfillmentOption {
   provider_id: string
   options: unknown[]
 }
 
+/**
+ * @schema AdminGetRegionsRegionFulfillmentOptionsRes
+ * type: object
+ * properties:
+ *   fulfillment_options:
+ *     type: array
+ *     items:
+ *       properties:
```

Comment:
missing `type: object`

Reply:
Fixed in 234b87f5571e0c4067cfb81f777396cb33903a64 86a5b071924e484d77febcb84949687e9e67028a